analyzer/src/test/kotlin/curation/FilePackageCurationProviderTest.kt

```diff
@@ -55,6 +55,24 @@ class FilePackageCurationProviderTest : StringSpec() {
             }
         }
 
+        "Provider can read from a single file and directories" {
+            val provider = FilePackageCurationProvider.from(curationsFile, curationsDir)
+
+            val idsWithExistingCurations = listOf(
+                // File
+                Identifier("npm", "", "ramda", "0.21.0"),
+                // Directory
+                Identifier("maven", "org.ossreviewtoolkit", "example", "1.0"),
+                Identifier("maven", "org.foo", "bar", "0.42")
+            )
+
+            idsWithExistingCurations.forEach {
+                val curations = provider.getCurationsFor(it)
+
+                curations should haveSize(1)
```

Comment:
hint: Maybe it suffices to just do `provider.packageCurations.size shouldBe x`

Reply:
I'd like to check for the IDs specifically so this test doesn't fail whenever a new test curation is added to the `curations.yml`.